GVRf/Framework/jni/objects/textures/cubemap_texture.h

```diff
@@ -0,0 +1,93 @@
+/* Copyright 2015 Samsung Electronics Co., LTD
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/***************************************************************************
+ * Cube map texture made by six bitmaps.
+ ***************************************************************************/
+
+#ifndef CUBEMAP_TEXTURE_H_
+#define CUBEMAP_TEXTURE_H_
+
+#include <string>
+
+#include <android/bitmap.h>
+
+#include "objects/textures/texture.h"
+#include "util/gvr_log.h"
+
+namespace gvr {
+class CubemapTexture: public Texture {
+public:
+	explicit CubemapTexture(JNIEnv* env, jobjectArray bitmapArray) :
```

Comment:
I moved env->GetArrayLength(bitmapArray) to cubemap_texture_jni.h.

For env->GetObjectArrayElement and AndroidBitmap_* functions, I think it is more efficient to be in cubemap_texture.h. If we move all of them to cubemap_texture_jni.h, we need to create six objects there and pass them to cubemap_texture.h, and delete the six objects then. This will introduce necessary memory allocation and delocation. I think it is the reason why base_texture.h has all AndroidBitmap_* functions in it, not in base_texture_jni.h.